docs-beta/content/quickstart/_index.md

```diff
@@ -0,0 +1,466 @@
+
+Placement initialization can take a minute or two. Once all the shards have the `AVAILABLE` state, the node has finished bootstrapping, and you should see the following messages in the node console output.
+
+<!-- TODO: Fix these timestamps -->
+
+```shell
+{"level":"info","ts":1598367624.0117292,"msg":"bootstrap marking all shards as bootstrapped","namespace":"default","namespace":"default","numShards":64}
+{"level":"info","ts":1598367624.0301404,"msg":"bootstrap index with bootstrapped index segments","namespace":"default","numIndexBlocks":0}
+{"level":"info","ts":1598367624.0301914,"msg":"bootstrap success","numShards":64,"bootstrapDuration":0.049208827}
+{"level":"info","ts":1598367624.03023,"msg":"bootstrapped"}
+```
+
+You can check on the status by calling the _placement_ endpoint:
+
+{{< tabs name="check_placement" >}}
+{{% tab name="Command" %}}
+
+```shell
+curl {{% apiendpoint %}}placement | jq .
+```
+
+{{% /tab %}}
+{{% tab name="Output" %}}
+
+```json
+{
+  "placement": {
+    "instances": {
+      "m3db_local": {
+        "id": "m3db_local",
+        "isolationGroup": "local",
+        "zone": "embedded",
+        "weight": 1,
+        "endpoint": "127.0.0.1:9000",
+        "shards": [
+          {
+            "id": 0,
+            "state": "AVAILABLE",
+            "sourceId": "",
+            "cutoverNanos": "0",
+            "cutoffNanos": "0"
+          },
+          …
+          {
+            "id": 63,
+            "state": "AVAILABLE",
+            "sourceId": "",
+            "cutoverNanos": "0",
+            "cutoffNanos": "0"
+          }
+        ],
+        "shardSetId": 0,
+        "hostname": "localhost",
+        "port": 9000,
+        "metadata": {
+          "debugPort": 0
+        }
+      }
+    },
+    "replicaFactor": 1,
+    "numShards": 64,
+    "isSharded": true,
+    "cutoverTime": "0",
+    "isMirrored": false,
+    "maxShardSetId": 0
+  },
+  "version": 2
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% notice tip %}}
+[Read more about the bootstrapping process](https://docs.m3db.io/operational_guide/bootstrapping_crash_recovery/).
+{{% /notice %}}
+
+## Writing and Querying Metrics
+
+M3DB supports two query engines:
+
+-   **Prometheus (default)** - robust and commonly-used query language for metrics
+-   **M3 Query Engine** - in development higher-performance query engine
+
+### Writing Metrics
+
+As M3DB is a time series database (TSDB), metric data consists of a value, a timestamp, and tags to bring context and meaning to the metric.
+
+You can write metrics using one of two endpoints:
+
+-   _[/prom/remote/write](/m3coordinator/api/remote/)_ - Write a Prometheus remote write query to M3DB with a binary snappy compressed Prometheus WriteRequest protobuf message.
+-   _/json/write_ - Write a JSON payload of metrics data.
+
+For this quickstart, use the _{{% apiendpoint %}}json/write_ endpoint to write a tagged metric to M3DB with the following data in the request body, all fields are required:
```

Comment:
Might be worth putting a warning here saying that the json write endpoint should only be used for debugging/testing purposes?

Reply:
@benraskin92 Sure, but a question. Why is it only for debugging or testing? Performance? Or just that it's more likely people will use protobuf message?

Reply:
Performance mainly.